src/test/java/de/hpi/bpt/argos/api/EntityTypeEndpointTest.java

```diff
@@ -82,11 +89,12 @@ public void testGetAttributes() {
     }
 
     @Test
-    public void testGetAttributes_InvalidId_BadRequest() {
+    public void testGetAttributes_InvalidId_Success() {
         RestRequest request = RestRequestFactoryImpl.getInstance()
                 .createGetRequest(ARGOS_REST_HOST, EntityTypeEndpoint.getEntityTypeAttributesUri(root1.getId() - 1));
 
-        assertEquals(HttpStatusCodes.BAD_REQUEST, request.getResponseCode());
+        assertEquals(HttpStatusCodes.SUCCESS, request.getResponseCode());

```

Comment:
In fact, I dont event know why the old version worked in first place...
All there is done is a database lookup, which will return an empty list at least. Therefore, the response code for this route will be always 200.